### ISSUE TITLE
Coordinates

### ISSUE THREAD

**Issue by Macgamut** [mtoensing/Docker-Minecraft-Bedrock-Server#42]
I can't for the life of me figure out how to change the settings file to turn on coordinates in the server. I've searched all of the documentation including the german tutorial and I can't see it anywhere. Help please?
*edit to add THANK you for developing this for our NAS. Your tutorial on Youtube was hugely helpful and we're all extremely pumped to have a server running on our own NAS for $0 per month. Phenomenal!

**Comment by Macgamut** [comment]
I got it figured out! 
For anyone else wondering and not aware of which command to use in the terminal, I found the answer in this guide.
https://support.aternos.org/hc/en-us/articles/360029388151-Show-coordinates-on-Bedrock-servers#h_e4efe945-f652-4596-b00a-a5e5c92ecf66

Just have to remove the "/" from the beginning of the command they show above. So to turn on coordinates, I put this into terminal for our new server:

gamerule showcoordinates true

And coordinates turned on!

Thanks again for an awesome way to run our own server on our Synology NAS. Much appreciated!
